extractors/bookseries/karelians/extractors/forest_extractor.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+import core.utils.regex_utils as regexUtils
+from core.pipeline_construction.base_extractor import BaseExtractor
+
+
+class ForestExtractor(BaseExtractor):
+    extraction_key = 'metsää'
+
+    def __init__(self, cursor_location_depends_on=None, options=None):
+        super(ForestExtractor, self).__init__(cursor_location_depends_on, options)
+        self.FOREST_PATTERN = r'(?:([0-9][0-9]) ha metsää){s<=1}|(?<!H)(a metsää){s<=1}|(metsää [0-9][0-9] ha){s<=1}'
+        self.FOREST_OPTIONS = (re.UNICODE | re.IGNORECASE)
+
+    def _extract(self, entry, extraction_results, extraction_metadata):
+        own_forest = self._find_forest(entry["text"])
+        return self._add_to_extraction_results(own_forest, extraction_results, extraction_metadata)
+
+    def _find_forest(self, text):

```

Comment:
Maybe `_find_forest_area`? Or `_find_if_owns_forest`? Wait, what was the spec actually, what was this extractor supposed to extract? Looking at the `_find_forest` function implementation, it looks like currently this extractor just extracts a `True`/`False` depending on whether ownership of forests is mentioned. Was that what was supposed to be extracted? Or was the area supposed to be extracted?